chat.simplex.app.yml

```diff
@@ -0,0 +1,54 @@
+app-id: chat.simplex.app
```

Comment:
`app` is really a poor choice as the application name.

Reply:
IMHO the app-id should be `chat.simplex.simplex`.

Reply:
Yeah, we can switch to that.